packages/@okta/i18n/package.json

```diff
@@ -82,20 +83,20 @@
   },
   "_requested": {
     "name": "@okta/i18n",
-    "raw": "@okta/i18n@35.2.0",
-    "rawSpec": "35.2.0",
+    "raw": "@okta/i18n@35.5.0",
+    "rawSpec": "35.5.0",
     "scope": "@okta",
-    "spec": "35.2.0",
+    "spec": "35.5.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#USER"
```

Comment:
I don't know why this happens, but as far as I know this should remain as ```/```, so we can use this package.

Reply:
Should I manually fix this to '/' ?

Reply:
I think so

Reply:
Cool.